testing/hexinject/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: Arch3y <arch3y@riseup.net>
+# Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
+pkgname=hexinject
+pkgver=1.5
+pkgrel=0
+pkgdesc="A very versatile packet injector and sniffer"
+url="http://hexinject.sourceforge.net/"
+arch="all"
+license="BSD"
+depends=""
+makedepends="libpcap-dev"
+source="https://downloads.sourceforge.net/$pkgname/$pkgname-$pkgver.tar.gz gcc5-fixes.patch"
+
+_builddir="$srcdir/$pkgname"
+
+prepare() {
+	local i
+	cd "$_builddir"
+	for i in $source; do
+		case $i in
+		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
+		esac
+	done
+}
+
+build() {
+	cd "$_builddir"
+	make
+}
+
+package() {
+	cd "$_builddir"
+        install -Dm755 hexinject $pkgdir/usr/bini/hexinject
```

Comment:
1) The indentation here seems off;
2) It'd be better to just use `cp` if the binaries already have the correct permissions;
3) Please quote `$pkgdir`;
4) Typo: `bini` -> `bin`;
5) Typo: `bin//` -> `bin/`;